content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -43,15 +43,13 @@ the appropriate snapshot.  For example, if you want to run with snapshot 6, use
 1.  Update Helm's local package cache with the location of the Helm daily release:
 
     {{< text bash >}}
-
     $ helm repo add istio.io "https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-1.1-latest-daily/charts/"
-
     {{< /text >}}
 
 1. Choose one of the following two **mutually exclusive** options described below.
 
-    - To use Kubernetes manifests to deploy Istio, follow the instructions for [option 1](/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template).
-    - To use [Helm's Tiller pod](https://helm.sh/) to manage your Istio release, follow the instructions for [option 2](/docs/setup/kubernetes/helm-install/#option-2-install-with-helm-and-tiller-via-helm-install).
+    * To use Kubernetes manifests to deploy Istio, follow the instructions for [option 1](/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template).
```

Comment:
What is the key difference here? Option 2 also uses manifests but through Tiller. To avoid confusion for the readers, be explicit:
```suggestion
    * To to deploy Istio without using Tiller, follow the instructions for [option 1](/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template).
```